Modules/dbupload.R

```diff
@@ -163,31 +163,87 @@ metric_mm_tm_Info_upload_to_DB <- function(package_name){
-                "'", package_name, "',", 
-                "'", package_riskmetric1$has_vignettes[1], ",", ifelse(class(package_riskmetric2$has_vignettes[[1]])[1] == "pkg_metric_error", -1, package_riskmetric2$has_vignettes[[1]][1]), "',",
-                "'", package_riskmetric1$has_news[1], ",",  ifelse(class(package_riskmetric2$has_news[[1]])[1] == "pkg_metric_error", -1, package_riskmetric2$has_news[[1]][1]), "',", 
-                "'", package_riskmetric1$news_current[1], ",",  ifelse(class(package_riskmetric2$news_current[[1]])[1] == "pkg_metric_error", -1, package_riskmetric2$news_current[[1]][1]), "',",  
-                "'", package_riskmetric1$has_website[1], ",",  ifelse(class(package_riskmetric2$has_website[[1]])[1] == "pkg_metric_error", -1, package_riskmetric2$has_website[[1]][1]), "',", 
-                "'", package_riskmetric1$has_bug_reports_url[1], ",",  ifelse(class(package_riskmetric2$has_bug_reports_url[[1]])[1] == "pkg_metric_error", -1, package_riskmetric2$has_bug_reports_url[[1]][1]), "',",
-                "'", package_riskmetric1$has_maintainer[1], ",",  ifelse(class(package_riskmetric2$has_maintainer[[1]])[1] == "pkg_metric_error", -1, package_riskmetric2$has_maintainer[[1]][1]), "',", 
-                "'", package_riskmetric1$has_source_control[1], ",",  ifelse(class(package_riskmetric2$has_source_control[[1]])[1] == "pkg_metric_error", -1, package_riskmetric2$has_source_control[[1]][1]), "',",
-                "'", format(round(package_riskmetric1$export_help[1],2)), ",",  ifelse(class(package_riskmetric2$export_help[[1]])[1] == "pkg_metric_error", -1, package_riskmetric2$export_help[[1]][1]), "',",
-                "'", format(round(package_riskmetric1$bugs_status[1],2)), ",",  ifelse(class(package_riskmetric2$bugs_status[[1]])[1] == "pkg_metric_error", -1, package_riskmetric2$bugs_status[[1]][1]), "'", ")"))
+  pkrm1 <- package_riskmetric1[- which(colnames(package_riskmetric1) %in% excl_name)]
+  pkrm2 <- package_riskmetric2[- which(colnames(package_riskmetric2) %in% excl_name)]
+  
+  # package_riskmetric1$bugs_status <- package_riskmetric1$bugs_status*100
+  # package_riskmetric1$export_help <- package_riskmetric1$export_help*100
+  
+  
+  # create tbl of names and labels
+  # colnames(pkrm3)
+  # vartibbl <- tibble(
+  #   names = c("news_current", "has_vignettes", "has_bug_reports_url",
+  #             "bugs_status",  "export_help",   "has_website",        
+  #             "has_maintainer", "has_news",     "has_source_control" ),
+  #   mm_label = c("News is current?","Presence of vignettes?","Bugs publicly documented?",
+  #                "Bug closure","Documentation","Associated website URL?",
+  #                "Has a maintainer?","Has NEWS?", "Source code public?")
+  # )
+  
+  metrics_to_read <- file.path("Data", "maint_metrics_labels.csv")
+  vartibbl <- readr::read_csv(metrics_to_read, col_types = cols(.default = "c"))
+  
+  # any label that does not have "?" at the end is assumed percentage
```

Comment:
The labels may change in the future, so relying on them may not be feasible. An approach to avoid this issue may be to add a new column indicating wether the metric is boolean.